lib/scheduler.registry.ts

```diff
@@ -8,6 +8,19 @@ export class SchedulerRegistry {
   private readonly timeouts = new Map<string, any>();
   private readonly intervals = new Map<string, any>();
 
+  isExists(type: 'cron' | 'timeout' | 'interval', name: string) {
```

Comment:
Personally, I like the name `doesExist` better, it sounds better in plain English, but there's nothing wrong otherwise. Seems like a good addition